src/coreclr/dlls/mscordac/CMakeLists.txt

```diff
@@ -57,18 +57,19 @@ else(CLR_CMAKE_HOST_WIN32)
         set(PAL_REDEFINES_INC ${GENERATED_INCLUDE_DIR}/palredefines.inc)
         add_custom_command(
             OUTPUT ${PAL_REDEFINES_INC}
-            COMMAND ${AWK} -f ${REDEFINES_FILE_AWK_SCRIPT} -v jump=${JUMP_INSTRUCTION} -v prefix2=DAC_ ${DEF_SOURCES} > ${PAL_REDEFINES_INC}
-            DEPENDS ${DEF_SOURCES} ${REDEFINES_FILE_AWK_SCRIPT}
+            COMMAND ${REDEFINES_FILE_SCRIPT} ${DEF_SOURCES} ${JUMP_INSTRUCTION} "" DAC_ > ${PAL_REDEFINES_INC}
+            DEPENDS ${DEF_SOURCES} ${REDEFINES_FILE_SCRIPT}
             COMMENT "Generating PAL redefines file -> ${PAL_REDEFINES_INC}"
+            VERBATIM
```

Comment:
This was required as we are passing an empty positional argument (`prefix1`). Without `VERBATIM` cmake's argument processor removes the empty quotes. (also available in [older cmake versions](https://cmake.org/cmake/help/v3.0/command/add_custom_command.html))